### PR TITLE
Update manifest to add new version|

### PR DESCRIPTION
**Pull request by nicholasjackson**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120004)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)  ---
> System.ComponentModel.Win32Exception (216): The specified executable is not a valid application for this OS platform. 
--- End of stack trace from previous location where exception was thrown ---    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)

File: manifests/j/jumppad/jumppad/0.5.49/jumppad.jumppad.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: jumppad.jumppad
+PackageVersion: 0.5.49
+Installers:
+- InstallerUrl: https://github.com/jumppad-labs/jumppad/releases/download/v0.5.49/jumppad_0.5.49_windows_x86_64.zip
+  Architecture: x86
```
Comment (stephengillie) [inline manifests/j/jumppad/jumppad/0.5.49/jumppad.jumppad.installer.yaml:8]:
```suggestion
  Architecture: x64
```
`x86` validations are performed on a 32-bit OS, so I'm guessing that this is a 64-bit program.

Comment (nicholasjackson) [reply manifests/j/jumppad/jumppad/0.5.49/jumppad.jumppad.installer.yaml:8]: Hey thanks, this is actually a mistake carried over from our first submission, will fix

**Comment by nicholasjackson** [comment]
Thanks @stephengillie for your suggested change, the automated pipeline that will handle future releases will ensure that this correct architecture is set.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**